src/modules/esl-carousel/plugin/nav/esl-carousel.nav.arrows.less

```diff
@@ -28,11 +24,9 @@
   // Positioning
   .esl-carousel-arrow {
     width: var(--esl-carousel-arrow-size);
-    height: 100%;
+    height: var(--esl-carousel-arrow-size);
```

Comment:
The only case we need to consider for consumers (not implement, we need just keep it in mind): 
There is a case where we want to have a clickable area bigger than the arrow. What should we suggest in that case?